Detectors/GlobalTracking/include/GlobalTracking/MatchTOF.h

```diff
@@ -269,6 +269,8 @@ class MatchTOF
 
   unsigned long mTimestamp = 0; ///< in ms
 
+  static unsigned long mFirstTForbit; ///< First orbit in TF (needed to align FT0 recpoints)
+
```

Comment:
Why this need to be static?

Reply:
It is needed since I kept findFITIndex method as static (I guess I am using it also in some external macros for debug purposes).
But if you consider it misleading I can put it as non-static and just passing it as an argument to that method.
Shall I do it?

Reply:
Done, let's see if at least one compilation check is passed (I didn't have the time since I am working on a different branch)